pools/kovan/metadata/0x6f3d561b203c69cdee90f2870ea58e5a7da460b2.json

```diff
@@ -0,0 +1,37 @@
+{
+  "env": "staging",
+  "startblock": 20911916,
+  "version": 2,
+  "addresses": {
+    "ROOT_CONTRACT": "0x6f3d561b203c69cdee90f2870ea58e5a7da460b2",
+    "TINLAKE_CURRENCY": "0xe689a4f88e332b016f6800259355e23ef6df338a",
+    "JUNIOR_OPERATOR": "0x4c74B9809FcdeE450cae69574e84aBA6e27b342b",
+    "JUNIOR_TRANCHE": "0xe854d6d6702db178279064099E00EbA30E660D96",
+    "JUNIOR_TOKEN": "0x21C0772953E3976458aBcF36A34F598b1dfE857B",
+    "SENIOR_TRANCHE": "0x33F521cb1dAeb96B1F7A99260eD6c2c68645FD97",
+    "SENIOR_TOKEN": "0x1acF690B6ca507c99a732486A52d66D2390bd599",
+    "SENIOR_OPERATOR": "0xd47daDE0299E7345d3905F364aE47BFD6305847A",
+    "DISTRIBUTOR": "0xe1859Fb897FE8A4956cebE2B7251fDe7737782aa",
+    "ASSESSOR": "0xB99B95e37EC8bc796FE8a18a8a61A3e36B007372",
+    "TITLE": "0xE7af9E965Be0eFd1C158ac8c7dE7ea8980D45B8a",
+    "PILE": "0x7A5D7331b7229014d0469dE5cDD369d119c7E4dE",
+    "SHELF": "0x838fAAE94Ed562a9597634d558987c7b5ACd6B84",
+    "CEILING": "0x66390B34772c8E3665ABa884b18c6365cEf091B8",
+    "COLLECTOR": "0x2d83E85426b8df12500b8eAf5C6B9c5818a47198",
+    "THRESHOLD": "0x66390B34772c8E3665ABa884b18c6365cEf091B8",
+    "PRICE_POOL": "0x7eCC1399bD9B07FC45cd50725734568aE34eB75D",
+    "NFT_FEED": "0x66390B34772c8E3665ABa884b18c6365cEf091B8",
+    "ACTIONS": "0x60cc4a2868559e513112f0742e6f546d339ea17b",
+    "PROXY_REGISTRY": "0xb0cd959bbbe799ba7d18c7d28008553a7b47a04c"
+  },
+  "contractConfig": {
+    "JUNIOR_OPERATOR": "ALLOWANCE_OPERATOR",
+    "SENIOR_OPERATOR": "PROPORTIONAL_OPERATOR"
+  },
+  "name": "Kovan Static NAV Pool 1",
+  "slug": "kovan-static-nav-pool-1",
+  "description": "<h4>Asset Originator Details</h4><p>This section displays information provided by the Asset Originator, such as a description of the business model</p><h4>Pool Details (Sample Data)</h4><p><strong>Asset Originator:</strong> Kovan NAV Pool 1 LLC<br /><strong>Collateral:</strong> Simplified invoices<br /><strong>Average Invoice Size:</strong> $1,000<br /><strong>Maturity Date:</strong> 30 day invoices<br /><strong>Issuer:</strong> Kovan NAV Pool 1 LLC</p>",
```

Comment:
Just noticed you used the old format. These were rewritten about a week ago. Maybe you can pull in the latest versions? See https://github.com/centrifuge/tinlake-pool-config/blob/master/kovan-staging.json (`description` is now replaced by various structured values and `investHtml` is removed).

Reply:
Fixed